Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -232,6 +233,11 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
     !!! note
         A value of 0 disables the quota.
 
+* `autoscaleMode`: Set autoscale mode to pool. See the [ceph documentation](https://docs.ceph.com/en/latest/rados/operations/placement-groups/) for more info.
+    * `off`: Disable autoscaling for this pool. It is up to the administrator to choose an appropriate pgp_num for each pool. Please refer to [Choosing the number of Placement Groups](https://docs.ceph.com/en/latest/rados/operations/placement-groups/#choosing-number-of-placement-groups) for more information.
+    * `on`: Enable automated adjustments of the PG count for the given pool.
+    * `warn`: Raise health alerts when the PG count should be adjusted.
+
```

Comment:
I find that it is best to also document the value that will implement defaulting behavior.

```suggestion
    * `` (empty): Use the default Ceph-configured pool autoscale strategy.

```